src/Main.java

```diff
@@ -1,7 +1,56 @@
+import java.util.Arrays;
+
 //TIP To <b>Run</b> code, press <shortcut actionId="Run"/> or
 // click the <icon src="AllIcons.Actions.Execute"/> icon in the gutter.
 public class Main {
     public static void main(String[] args) {
 
+        //Task#1
+        int [] array = {5000, 3500, 4200, 3700, 2900};
+        int count = 0;
+        for (int i = 0; i < array.length; i++) {
+            count += array[i];
+        }
+        System.out.println("Сумма трат за меся составила " + count + " рублей");
+
+        System.out.println();
+        //Task#2
+
+        int minimalValue = 99999;
```

Comment:
здесь в качестве начального значения для минимума правильнее взять максимальное число для int, потому что если в массиве будет число больше 99999, то минимум будет найден неверно
```suggestion
        int minimalValue = Integer.MAX_VALUE;
```